src/pages/ProfileCloseAcountPage.jsx

```diff
@@ -0,0 +1,45 @@
+import ButtonProfile from "../shared/ButtonProfile";
+import ProfileHeader from "../shared/ProfileHeader";
+
+const ProfileCloseAcountPage = () => {
+  const borderStyle = "border border-gray-400";
+  const title = "Close Account";
+  const description = "Close your account permanently.";
+  return (
+    <div>
+      <section className={`py-8 border-b-0 ${borderStyle}`}>
+        <ProfileHeader title={title} description={description} />
+        <div className="px-3 pb-12 border-b border-gray-400 md:px-5 lg:px-10">
+          <div className="flex flex-wrap justify-between gap-5 py-5">
+            <p className="text-l md:text-xl">
+              Suspending account activity for an indefinite period
+            </p>
+
+            <select
+              name=""
+              id=""
+              className={`lg:px-4 lg:py-1 outline-none ${borderStyle}`}
+            >
+              <option value="">Select period</option>
```

Comment:
It's better to render the options automatically, you can store the options strings in an array, then map on it, also you need to add a value to the value property.

notice that if this is a dropdown menu, it's better in React.js to render it with (ul & li) elements not (select & option) elements, we already have a dropdown model in the shared folder, check it if you prefer to use it.